src/EditorComponent.jsx

```diff
@@ -163,6 +202,53 @@ const EditorComponent = props => {
             return true;
         };
 
+        // Editor command that simply tells the extension to capture the current frame. Does not apply any
+        // transaction to the state.
+        const tellExtensionToCaptureFrame = (state, dispatch) => {
+            const { videoId } = parentApp.currentVideoInfo();
+
+            if (!videoId) {
+                return false;
+            }
+
+            window.frames[0].postMessage({ type: AppConfig.CaptureCurrentFrameMessage }, '*');
+
+            return true;
```

Comment:
Why do we need to return a boolean from this function?

Reply:
Since this is a prosemirror-command, if you return true it says it's handled by you, and false means it's not handled. Since we don't care, I think I should just return true.